src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import command.AddDeadlineCommand;
+import command.AddEventCommand;
+import command.AddToDoCommand;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.MarkDoneCommand;
+import command.TaskListCommand;
+import command.UnknownCommand;
+
+public class ParserTest {
+    @Test
+    public void parseAddDeadline() {
```

Comment:
Not a requirement for test code, but it is recommended to write in tests in the following format -> whatIsBeingTested_descriptionOfTestInputs_expectedOutcome

Rationale is that when you see errors occurring due to your unit tests, it would be good to be able to trace exact test case scenarios and narrow down the scope of why errors are occurring. It would be particularly important in the case of regression testing. Once code bases grow larger, it would be easier for other developers who are working on the code together to realise what problems are being created when they refactor the code without prior knowledge in an effort to introduce new features.

Reply:
I see, thanks for the suggestion!